mne/viz/ica.py

```diff
@@ -130,6 +130,264 @@ def plot_ica_sources(ica, inst, picks=None, exclude=None, start=None,
     return fig
 
 
+def _create_properties_layout(figsize=None):
+    """creates main figure and axes layout used by plot_ica_properties"""
+    import matplotlib.pyplot as plt
+    if figsize is None:
+        figsize = [7., 6.]
+    fig = plt.figure(figsize=figsize, facecolor=[0.95] * 3)
+    ax = list()
+    ax.append(fig.add_axes([0.08, 0.5, 0.3, 0.45], label='topomap'))
+    ax.append(fig.add_axes([0.5, 0.6, 0.45, 0.35], label='image'))
+    ax.append(fig.add_axes([0.5, 0.5, 0.45, 0.1], label='erp'))
+    ax.append(fig.add_axes([0.08, 0.1, 0.32, 0.3], label='spectrum'))
+    ax.append(fig.add_axes([0.5, 0.1, 0.45, 0.25], label='variance'))
+    return fig, ax
+
+
+def plot_properties(inst, ica=None, picks=None, axes=None, dB=True, cmap=None,
```

Comment:
It's for channel properties (if no ica is passed picks are treated as channel indices and channel properties are displayed), but I can remove it here and go back to it when doing channel properties PR.